src/components/filter.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+enum FilterType {
```

Comment:
do not duplicate types

instead, move it to a separate file and export them. Import in files where you need them